promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,105 @@
+# Connect Promscale and Grafana
+Grafana ships with built-in Prometheus, PostgreSQL, and other data source
+plugins that allow you to query and visualize data from a compatible database.
+To add a data source in Grafana you must be signed as a user with organization 
+administration role privileges.
+
+To connect Grafana with Promscale, install and run Grafana version 5.3 or later.
+For information about installing Grafana, see the
+[Grafana installation documentation][grafana-install].
+
+This section shows you how to connect Promscale to Prometheus, Jaeger, and
+PostgreSQL data sources in [Grafana][grafana-homepage].
+
+<procedure>
+
+## Configuring Promscale as Prometheus data source
+
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Prometheus` in the search field and click `Select`.
+1.  Configure the data source:
+    *   In the `Name`field, type `Promscale-metrics`.
+    *   In the `URL` field, type `http://<PROMSCALE-IP-ADDR>:9201`, where
+        `<PROMSCALE-IP-ADDR>` is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+When you have configured Promscale as a Prometheus data source in Grafana, you
+can create panels that are populated with data using PromQL.
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/images/misc/getting-started-with-promscale-grafana-dashboard.png"
+alt="Sample output for PromQl query"/>
+
+Grafana ships with built-in support for Jaeger, which provides open source,
```

Comment:
Shouldn't this be part of the Configuring Promscale as Jaeger data source section? I understand we use a procedure but it's weird to see this text directly after talking about the Prometheus data source with not break in between (some title usually).